test/api/controllers/remoteActionsController/prepareDb.test.js

```diff
@@ -306,12 +304,10 @@ describe('Test: Prepare database', function () {
       importMapping,
       workerCalled,
       mappingCreated,
-      workerPromise,
-      prepareDb;
+      workerPromise;
 
     before(function () {
-      prepareDb = rewire('../../../../lib/api/controllers/remoteActions/prepareDb');
-      importMapping = prepareDb.__get__('importMapping');
+      importMapping = rewire('../../../../lib/api/controllers/remoteActions/prepareDb').__get__('importMapping');
```

Comment:
why not sharing the same 'prepareDb' in the whole test file ? (or at least use a global variable for the filename ?)